models/projects.js

```diff
@@ -207,6 +207,37 @@ async function deleteTask(taskId) {
   }
 }
 
+//Teams
```

Comment:
You can throw all this into a new file called `teams.js` since it's separate logic from projects.

Reply:
Copy, I will do that. Was unsure if that was the direction you wanted to go or not.